### ISSUE TITLE
Unable to open the repository in Visual Studio Code

### ISSUE THREAD

**Issue by Akshit3010** [desktop/desktop#14996]
### The problem

Error: spawn C:\Windows\system32\cmd.exe ENOENT
    at __node_internal_captureLargerStackTrace (node:internal/errors:464:5)
    at __node_internal_errnoException (node:internal/errors:594:12)
    at Process.ChildProcess._handle.onexit (node:internal/child_process:282:19)
    at onErrorNT (node:internal/child_process:477:16)
    at processTicksAndRejections (node:internal/process/task_queues:83:21)

### Release version

3.0.5

### Operating system

Windows 8.1

**Comment by sergiou87** [comment]
Thanks for your report @Akshit3010 

The repository you're trying to open… is somewhere in C:/ or is it in a different drive?

**Comment by Akshit3010** [comment]
@sergiou87 No, it is in D:/ Drive. I also set ComSpec to %SystemRoot%\system32\cmd.exe in the environment variables but still facing the same issue.

**Comment by sergiou87** [comment]
@Akshit3010 could you try moving/copying the repository to the C:/ drive, re-add it to GitHub Desktop, and see if you can then open it on VS Code?

I'm just wondering if that has anything to do with your issue 🤔

**Comment by sergiou87** [comment]
A few more questions: Are you in a machine with different users? Is your user an admin on that machine? Do you have access to `C:/Windows/System32/cmd.exe`?

**Comment by Akshit3010** [comment]
Got the issue, actually I was using my friend's machine and yes the problem was with authorization. I haven't had the admin access. Thanks @sergiou87

**Comment by sergiou87** [comment]
Thank you for confirming @Akshit3010 🙇‍♂️ 

Just curious: how was your friend's machine set up to reproduce this behavior? I tried creating a guest user on Windows but it worked fine, so I'm wondering how exactly was the authorization restricted in your case.

**Comment by Akshit3010** [comment]
Actually, my friend forgot his main account's password and lost his access to the main account and now using his machine as a guest user. So when I installed Github Desktop, there was some issue with setting the env variables as I was not authorized to set them for admin and in guest mode, the desktop kept giving the same error. Then I searched on StackOverflow and found one comment stating change your ComSpec to %SystemRoot%\system32\cmd.exe but was unable to do that as there was no option for changing the ComSpec in guest mode.
